submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts

```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *   http://www.apache.org/licenses/LICENSE-2.0
+ *   http: //www.apache.org/licenses/LICENSE-2.0
```

Comment:
This space is not needed actually. Maybe some settings in your IDE cause this?

Reply:
Thanks. I've fixed it.